src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/DataTableCaption.test.tsx

```diff
@@ -0,0 +1,338 @@
+import React from 'react';
+import {
+  CategoryFilter,
+  Indicator,
+  LocationFilter,
+  TimePeriodFilter,
+} from '@common/modules/table-tool/types/filters';
+import { FullTableMeta } from '@common/modules/table-tool/types/fullTable';
+import { render, screen, fireEvent } from '@testing-library/react';
+import DataTableCaption from '../DataTableCaption';
+
+describe('DataTableCaption', () => {
+  const testfullTableMeta: FullTableMeta = {
+    geoJsonAvailable: false,
+    publicationName: 'Pupil absence in schools in England',
+    subjectName: 'Absence by characteristic',
+    footnotes: [],
+    boundaryLevels: [],
+    filters: {
+      Characteristic: {
+        name: 'characteristic',
+        options: [
+          new CategoryFilter({
+            value: 'total',
+            label: 'Total',
+            group: 'Gender',
+            category: 'Characteristic',
+          }),
+        ],
+      },
+    },
+    indicators: [
+      new Indicator({
+        label: 'Authorised absence rate',
+        value: 'authAbsRate',
+        unit: '%',
+        name: 'sess_authorised_percent',
+      }),
+    ],
+    locations: [
+      new LocationFilter({
+        value: 'E92000001',
+        label: 'England',
+        level: 'country',
+      }),
+    ],
+    timePeriodRange: [
+      new TimePeriodFilter({
+        code: 'AY',
+        year: 2015,
+        label: '2015/16',
+        order: 0,
+      }),
+    ],
+  };
+
+  test('with given title', () => {
+    render(<DataTableCaption title="Test title" {...testfullTableMeta} />);
+
+    expect(screen.queryByText('Test title')).not.toBeNull();
+    expect(
+      screen.queryByRole('button', { name: /full table title/ }),
+    ).toBeNull();
+  });
+
+  test('with multiple filters labelled "Total"', () => {
+    const props = {
```

Comment:
Add explicit type. Whenever you're creating new plain JSON objects like this in tests, make sure they're typed otherwise it makes it easy for incorrect properties to sneak in.